samples/openthread/cli/overlay-multiprotocol.conf

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Increase stack
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2144
```

Comment:
I'm wondering if it this is really needed for CLI sample. If it is needed and it is multiprotocol dependent it should be applied in general subsys/openthread when CONFIG_BT is being enabled.